src/current/_includes/releases/v23.1/v23.1.8.md

```diff
@@ -0,0 +1,35 @@
+## v23.1.8
+
+Release Date: August 7, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-1-8-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- [Azure Blob Storage credentials for implicit authentication](../v23.1/cloud-storage-authentication.html?filters=azure#azure-blob-storage-implicit-authentication) can now be stored in a file with the following YAML format:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ yaml
+    azure_tenant_id: {MY_TENANT_ID},
+    azure_client_id: {MY_CLIENT_ID},
+    azure_client_secret: {MY_CLIENT_SECRET}
+    ~~~
+
+    Set the environment variable `COCKROACH_AZURE_APPLICATION_CREDENTIALS_FILE` to the path of the file. The implicit authentication method first attempts to authenticate using credentials in the file before attempting to authenticate using any other credentials specified in the [`DefaultAzureCredential` class](https://learn.microsoft.com/en-us/dotnet/api/azure.identity.defaultazurecredential?view=azure-dotnet), such as environment variables and Managed Identity. These credentials can also be used for [encrypted backups](../v23.1/take-and-restore-encrypted-backups.html#use-key-management-service). [#106688][#106688]
```

Comment:
```suggestion
    Set the environment variable `COCKROACH_AZURE_APPLICATION_CREDENTIALS_FILE` to the path of the file. The implicit authentication method first attempts to authenticate using credentials in the file before attempting to authenticate using any other credentials specified in the [`DefaultAzureCredential` class](https://learn.microsoft.com/dotnet/api/azure.identity.defaultazurecredential?view=azure-dotnet), such as environment variables and Managed Identity. These credentials can also be used for [encrypted backups](../v23.1/take-and-restore-encrypted-backups.html#use-key-management-service). [#106688][#106688]
```
Remove the localized portion so that the reader's default locale is respected for localized pages on microsoft.com.